### PR TITLE
Added GA event tracking to a footer link

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #1700

QA instructions

1. make sure you have "Do Not Track" browser setting turned off
2. make sure you disable privacy related add-ons
3. open [GA Dashboard](https://analytics.google.com/analytics/web/#/realtime/rt-event/a87658599w138395144p152070124/filter.list=41==page%252520footer%252520cta;40==navigation;&metric.type=5//) in a new tab
4. visit the following test links and click on the "See more" link near footer section
 screen shot 2018-08-13 at 3 08 12 pm

- https://foundation-mofostaging-pr-1740.herokuapp.com/campaigns/indeed-grow-health/
- https://foundation-mofostaging-pr-1740.herokuapp.com/campaigns/them-stand-sit/
- https://foundation-mofostaging-pr-1740.herokuapp.com/campaigns/interest-find-near/

5. check GA Dashboard and see if the action has been recorded

example screencap:
screen shot 2018-08-13 at 3 09 41 pm

### CODE REVIEW

**Comment by mmmavis** [comment]
@xmatthewx not sure if i should assign you or Tais for design review.

**Comment by xmatthewx** [comment]
@taisdesouzalessa - give this a whirl. ping me on slack if you don't have enough info.

**Comment by taisdesouzalessa** [comment]
It seems I don't have permission to see this view:
image

@cadecairos is this something you could help me with?

**Comment by xmatthewx** [comment]
Try it now @taisdesouzalessa. I had Alan add you. Let us know if it works.

**Comment by taisdesouzalessa** [comment]
Tested on desktop (Firefox and Chrome) and mobile (Firefox). It works great! \0/

File: source/js/main.js [hunk 1]
```diff
@@ -186,6 +190,19 @@ let main = {
     });
   },
 
+  bindGAEventTrackers() {
+    document.querySelector(`#see-more-modular-page`).addEventListener(`click`, () => {
+      let label = ``;
+      let pageHeader = document.querySelectorAll(`.cms h1`)[0];
```
Comment (gvn) [inline source/js/main.js]: You can just use `querySelector` if you only want the first result.

File: source/js/main.js [hunk 2]
```diff
@@ -186,6 +190,19 @@ let main = {
...
+
+      if (pageHeader) {
+        label = pageHeader.innerText + ` - footer cta`;
```
Comment (gvn) [inline source/js/main.js]: With template strings you shouldn't need to use concatenation anymore, just:

```
`${pageHeader.innerText} - footer cta`
```

File: source/js/main.js (hunk 1 above)
Comment (mmmavis) [reply source/js/main.js]: oh yea apparently i wasn't thinking 🤦‍♂️

File: source/js/main.js (hunk 2 above)
Comment (mmmavis) [reply source/js/main.js]: that was straight up c/p from my previous code . will update 😂

**Pull request merged**